File: swift/Husband.swift
class Husband {
    var name: String
    var wife: Wife?

    init(name: String, wife: Wife?) {
        self.name = name
        self.wife = wife
    }

    func getMarried(to wife: Wife) {
        self.wife = wife
    }

    func getSalutation() -> String {
        if let wife = self.wife {
            return "Mr. and Mrs. \(name)"
        } else {
            return "Mr. \(name)"
        }
    }
}

File: python/husband.py
class Husband:
    def __init__(self, name):
        self.name = name
        self.wife = None

    def getMarried(self, wife):
        self.wife = wife

    def getSalutation(self):
        if self.wife:
            return f"Mr. and Mrs. {self.name}"
        else:
            return f"Mr. {self.name}"
